### PR TITLE
Avoid deserialize error cause Scene to be erased

### PR DESCRIPTION
**Pull request by pandamicro**
> Makes sure these boxes are checked before submitting your PR - thank you!
> - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
> - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)
> - [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
> - [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.

Re: cocos-creator/fireball#2197

Changes proposed in this pull request:
- 避免 deserialize 过程的错误导致场景数据丢失

@cocos-creator/engine-admins

### CODE REVIEW

File: cocos2d/audio/CCAudio.js [hunk 1]
```diff
@@ -108,9 +108,9 @@ JS.mixin(cc.Audio.prototype, {
     },
 
     getPlaying: function(){
-        if(!this._playing){
-            return false;
-        }
+        // if(!this._playing){
+        //     return false;
+        // }
```
Comment (pandamicro) [inline cocos2d/audio/CCAudio.js:113]: 修复在部分手机浏览器上 playing 状态无法改变的问题

File: cocos2d/core/load-pipeline/uuid-loader.js [hunk 2]
```diff
@@ -67,10 +67,17 @@ function loadUuid (item, callback) {
...
+    try {
+        asset = cc.deserialize(json, tdInfo, {
+            classFinder: classFinder,
+            target: item.existingAsset
+        });
+    }
+    catch (e) {
+        callback( new Error('Uuid Loader: Deserialize asset [' + item.src + '] failed : ' + e.stack) );
+        return;
+    }
```
Comment (jareguo) [inline cocos2d/core/load-pipeline/uuid-loader.js]: 使用了 try catch 以后，V8 将无法对整个方法启用高效优化。如果当前方法的代码比较多，建议把 try catch 和里面的东西移出去单独作为一个函数。

**Comment by jareguo** [comment]
这个和 #429 不是同一个问题……
有关联的修复是 https://github.com/cocos-creator/fireball/pull/2192

File: cocos2d/core/load-pipeline/uuid-loader.js (hunk 2 above)
Comment (pandamicro) [inline cocos2d/core/load-pipeline/uuid-loader.js]: 用 CC_EDITOR 隔离是不是在浏览器运行时就不会受影响了

Comment (jareguo) [inline cocos2d/core/load-pipeline/uuid-loader.js]: 对

**Pull request merged**